### ISSUE TITLE
whether Intuitor can work on not reasoning sceanrio

### ISSUE THREAD

**Issue by FWXT** [sunblaze-ucb/Intuitor#9]
From the paper, Intuitor can improve accuracy on reasoning sceanrio. But I need to inference in time without reasoning cot. Whether Intuitor can work on such scearnio and improve answer accuracy. Thx!

**Comment by XuandongZhao** [comment]
That's a great question. We ran some experiments on the MMLU-pro AlpacaEval benchmark, and the model showed improved performance compared to the base model. However, we're not sure how well it generalizes to other scenarios.

If you're interested, you could test the performance of [Qwen2.5-3B-Intuitor-MATH-1EPOCH](https://huggingface.co/sunblaze-ucb/Qwen2.5-3B-Intuitor-MATH-1EPOCH) versus the base model [Qwen2.5-3B](https://huggingface.co/Qwen/Qwen2.5-3B) on your specific tasks. If you see better results, you might even consider preparing additional training data to fine-tune a new version of Intuitor, since the current model on Hugging Face was only trained on the MATH dataset for one epoch.
